src/views/tutorial-view/components/collection-card/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import CardLink from 'components/card-link'
+import classNames from 'classnames'
+import { CollectionCardProps } from './types'
+import s from './collection-card.module.css'
+
+/**
+ * TODO: styles and functionality dependent on design spec being finalized.
+ * (currently "WIP" in Figma)
+ * Figma: https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6860%3A79233
+ */
+function CollectionCard({
+  className,
+  slug,
+  numTutorials,
+  name,
+  description,
+  theme,
+}: CollectionCardProps) {
+  return (
+    <CardLink
+      className={classNames(s.root, className)}
+      href={getCollectionSlug(slug)}
```

Comment:
Another note here is that some links will not work for product collections that haven't been added to dev dot yet. [Asana task here](https://app.asana.com/0/1201987349274776/1201991101510148) to address that

Reply:
Great callout - added a comment that links to that task 👍